microk8s-resources/wrappers/microk8s-add-node.wrapper

```diff
@@ -37,11 +37,8 @@ exit_if_stopped
 # We throw away some randomness by only selecting bytes in the A-Za-z range and taking the first 32 such bytes,
 # so instead of 32*8 = 256 bits of randomness you get approx 182 bits. This is good enough for this short lived tokens.
 token=$(< /dev/urandom tr -dc A-Za-z | head -c32;echo;)
-echo ${token} >> $SNAP_DATA/credentials/cluster-tokens.txt
-if getent group microk8s >/dev/null 2>&1
-then
-  chgrp microk8s $SNAP_DATA/credentials/cluster-tokens.txt || true
-fi
+# Call add-token wrapper to add it to the cluster-tokens.txt file.
+$SNAP/microk8s-add-token.wrapper --token $token --ttl 3600
```

Comment:
Instead of setting the token to always `--ttl 3600` let's keep the old default in the `add-node`  command.

We could have:
```
microk8s.add-node # will give you a one time token
microk8s.add-node --ttl-token 4000 # will give you a token that expires after 4000 seconds
microk8s.add-node --multinode-token 40 # will give you a token that can be used 40 times
...
```

Reply:
Interesting feature, instead of time to live, its based on number of nodes.  Didn't thought about that.
Let me see how to implement that.